pkg/db/schema/runtime/V0_1__init.sql.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE runtime_env (
+	runtime_env_id  VARCHAR(50) PRIMARY KEY             NOT NULL,
+	name            VARCHAR(50)                         NOT NULL,
+	description     TEXT                                NOT NULL,
+	runtime_env_url TEXT                                NOT NULL,
+	owner           VARCHAR(50)                         NOT NULL,
+	status          VARCHAR(50)                         NOT NULL,
+	create_time     TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
+	status_time     TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL
+);
+
+CREATE TABLE runtime_env_label (
+	runtime_env_id VARCHAR(50) NOT NULL,
```

Comment:
There should be used `runtime_env_label_id` as PRIMARY KEY.

Reply:
If I use runtime_env_id + Label_key as PRIMARY key is reasonable ?